code-server/run.sh

```diff
@@ -70,4 +70,24 @@ for extension in "$${EXTENSIONLIST[@]}"; do
   fi
 done
 
+if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
+  WORKSPACE_DIR="$HOME/"
```

Comment:
With the `/` here we could get a double slash below.  I think generally this works fine, although maybe there are cases where it does not?

Reply:
cool. i will look into that.